### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Harrrinator** [community/community#40936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How do i translate spanish into english on Python

**Comment by ossanna16** [comment]
Hi @Harrrinator,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Python. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
